bigtable/integration_test.go

```diff
@@ -1263,14 +1265,25 @@ func TestIntegration_Admin(t *testing.T) {
 		t.Errorf("Invalid row count after truncating table: got %v, want %v", gotRowCount, 0)
 	}
 
-	// Validate Encyrption Info not configured
-	encryptionInfo, err := adminClient.EncryptionInfo(ctx, "mytable")
-	if err != nil {
-		t.Fatalf("EncryptionInfo: %v", err)
-	}
-	if got, want := len(encryptionInfo), 0; !cmp.Equal(got, want) {
-		t.Fatalf("Number of Clusters with Encryption Info: %v, want: %v", got, want)
+	// Validate Encryption Info configured to default. (not supported by emulator)
+	if testEnv.Config().UseProd {
```

Comment:
Is it possible to set up the tests so that we can separate out ones that need prod and ones that don't? I feel like this  setup where we skip part of a test is not great.

Reply:
I agree. I am going to follow this with a PR that also allows using env vars, so we can add a prod targeted IT run to continuous


It isn't really an either or, but there are some tests the emulator doesn't support.